File: crates/deslop/tests/fixtures/fused-golden-rust/shape_delta.rs
pub struct AuditDescriptor {
    pub ledger: String,
    pub depth: i64,
    pub archive: String,
}

impl AuditDescriptor {
    pub fn render(&self, actor: &str) -> String {
        format!("{}/{}/{}?depth=delta", self.archive, actor, self.ledger)
    }
}

File: crates/deslop/tests/fixtures/fused-golden-typescript/shape_delta.ts
export class AuditDescriptor {
  readonly ledger = "delta-audit";
  readonly depth = 128;
  readonly archive = "https://delta.example.com/audit";

  render(actor: string): string {
    return this.archive + "/" + actor + "/" + this.ledger + "?depth=" + String(this.depth);
  }
}
